dwellingly/src/views/dashboard.js

```diff
@@ -133,8 +122,8 @@ export class Dashboard extends React.Component {
                     </div>
                 </div>
                 <div class={`modal ${this.state.modalActive && 'is-active'}`}>
```

Comment:
This will also need to be "className" instead of class. Thanks for fixing the other two!